app/means_test/api.py

```diff
@@ -22,3 +43,148 @@ def is_eligible(reference):
     means_test_endpoint = "checker/api/v1/eligibility_check/"
     response = cla_backend.post(f"{means_test_endpoint}{reference}/is_eligible/")
     return response["is_eligible"]
+
+
+def get_means_test_payload(eligibility_data: EligibilityData) -> dict:
```

Comment:
NON-BLOCKING: I think this function is has one too many ways adding payload to the `payload` dict, we should pick one and update this function.